src/Event/VendorEvent.php

```diff
@@ -42,6 +43,17 @@ public function __construct(string $type, array $identifiers, string $identifier
         $this->vendorId = $vendorId;
     }
 
+    /**
+     * Change the current state.
+     *
+     * @param $newVendorState
+     *   Change the state to this state. Please use the static properties found in the VendorState class
+     */
+    public function changeVendorState($newVendorState): void
+    {
+        $this->type = $newVendorState;
```

Comment:
type = state ??

Reply:
Yes, it's technically the state which is also the type of the event...

Reply:
Fixed